src/main/java/mod/gate/Gate.java

```diff
@@ -24,13 +25,19 @@ public class Gate implements ClientModInitializer {
     public void onInitializeClient() {
        LOGGER.info(Reference.NAME+" has been initialized!");
 
+       //load config
+        initConfig();
+    }
+
+    public void initConfig() {
```

Comment:
Since the way config works will be changed by me, there is no need to name a function for this, Just put the content of the function there